### PR TITLE
Drop master-stable, deploy stage-stable from master conditionally

### PR DESCRIPTION
**Pull request by himdel**
Follow-up to https://github.com/ansible/ansible-hub-ui/pull/1770

Before:
* cloud-stage-cron periodically pushes master to master-stable when $flag
* master-stable push triggers stage-stable deploy (but the cron one actually doesn't because of actions not triggering actions)
* master push triggers stage-beta deploy
* cloud-stage-{enable,disable} set $flag

After:
* master push triggers stage-beta deploy
* master push ALSO triggers stage-stable deploy *when $flag*
* cloud-stage-{enable,disable} set $flag
(no master-stable branch, no cloud-stage-cron action)

This means stage-stable updates together with stage-beta unless paused,
but it also means that it won't update when unpaused until a new PR is merged (but `deploy-cloud` can still be triggered manually)

### CODE REVIEW

File: .travis/custom_release.sh [hunk 1]
```diff
@@ -2,21 +2,16 @@
...
 set -x
 
-if [ "${TRAVIS_BRANCH}" = "master" ]
-then
-    for env in ci qa
-    do
+if [ "${TRAVIS_BRANCH}" = "master" ]; then
+    for env in ci qa ; do
+        # always push to stage-beta
         echo "PUSHING ${env}-beta"
```
Comment (awcrosby) [inline .travis/custom_release.sh:8]: What are the values of `ci` and `qa` on a push to the `master` branch? Are they different values when pushing to other branches?

**Review by awcrosby — APPROVED** [review]
I think this looks pretty good, i added a minor question.

File: README.md [hunk 2]
```diff
@@ -58,9 +58,8 @@ The Github Action invokes the [RedHatInsights/insights-frontend-builder-common//
 List of all workflows:
 
 - `backported-labels`: Add a backported-* label when a PR is backported to stable-*; on patchback merges
-- `cloud-stage-cron`: Deploy master to master-stable when enabled, trigger `deploy-cloud`; cron
-- `cloud-stage-disable`: Disable cron deploy from master; manual
-- `cloud-stage-enable`: Enable cron deploy from master; manual
+- `cloud-stage-disable`: Disable deploy-cloud from master to stage-stable (stage-beta always on); manual
+- `cloud-stage-enable`: Enable deploy-cloud from master to stage-stable (stage-beta always on); manual
```
Comment (awcrosby) [inline README.md:62]: Can we also rename the workflows? `Enable periodic stage deploy` `Disable periodic stage deploy` s/periodic/automatic

Comment (himdel) [reply README.md:62]: Updated :)

**Review by awcrosby — APPROVED** [review]

**Pull request merged**